mautic/mautic
# Dashboard widget -  best email read hours

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Added new dashboard widget with email best read performing hours. 
Available option to enable 12 and 24 hour time format in graph

![image](https://user-images.githubusercontent.com/462477/53126315-237c6c00-3560-11e9-9440-07f0d396efc4.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new dashboard widget Best email read hours
3. See If works properly with email read stats
